caps/capabilities.go

```diff
@@ -92,8 +97,14 @@ func (r *Repository) Add(cap *Capability) error {
 
 // Remove removes the capability with the provided name.
 // Removing a capability that doesn't exist silently does nothing
-func (r *Repository) Remove(name string) {
-	delete(r.caps, name)
+func (r *Repository) Remove(name string) error {
```

Comment:
This is driven by the REST API where we'd like to know if remove failed and there can be more than one reason why it can fail.